presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -223,7 +225,7 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
             reprintLine(perNodeSummary);
 
             // Parallelism: 5.3
-            out.println(format("Parallelism: %.1f", parallelism));
+            out.printf("Parallelism: %.1f%n", parallelism);
```

Comment:
Unrelated change?

Reply:
My IDE made this change and I forgot to undo it.